spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/serializer/ModuleSerializerSpec.scala

```diff
@@ -2058,6 +2058,20 @@ class ModuleSerializerSpec extends FlatSpec with Matchers {
     loadedModel.isTraining() should be (false)
   }
 
+  "HardSigmoid serialization" should "work properly" in {
+    val hardSigmoid = HardSigmoid()
```

Comment:
Wondering should we expose this as a generic test and put it into `beforeEach` for every `Layer` test? `BigDLSpecHelper` or `BigDLLayerSpec` would be a good place for this.

Reply:
@yiheng any suggestion? I think torchspec would like this as well.